sklearn/linear_model/tests/test_base.py

```diff
@@ -399,7 +405,8 @@ def test_preprocess_data():
     assert_array_almost_equal(yt, y - expected_y_mean)
 
 
-def test_preprocess_data_multioutput():
+def test_preprocess_data_multioutput(global_random_seed):
+    rng = check_random_state(global_random_seed)
```

Comment:
updated for consistency: `check_random_state` is used in some functions, but not all

Reply:
```suggestion
    rng = np.random.RandomState(global_random_seed)
```